quickstart/azure/setup.md

```diff
@@ -92,17 +94,31 @@ For example, a common Service Principal as displayed by the Azure CLI looks some
 }
 ```
 
+However, you also need to obtain subscription id. To get current subscription Id you can use:
+
+```
+az account show --query id -o tsv
+```
+
+To list available subscriptions use:
+
+```
+az account list --query '[].{subscriptionName:name,subscriptionId:id}' -o tsv
+```
+
 The environment variables would then be set as such:
 
 ```bash
-$ export ARM_CLIENT_ID="XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
-$ export ARM_CLIENT_SECRET="YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY"
-$ export ARM_TENANT_ID="ZZZZZZZZ-ZZZZ-ZZZZ-ZZZZ-ZZZZZZZZZZZZ"
+$ export ARM_CLIENT_ID="WWWWWWWW-WWWW-WWWW-WWWW-WWWWWWWWWWWW"
+$ export ARM_CLIENT_SECRET="XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+$ export ARM_TENANT_ID="YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY"
+$ export ARM_SUBSCRIPTION_ID="ZZZZZZZZ-ZZZZ-ZZZZ-ZZZZ-ZZZZZZZZZZZZ"
 ```
 
 Or configuration variables, if you prefer that they be stored alongside your Pulumi stack for easy multi-user access:
 
 ```bash
+$ pulumi config set azure:subscriptionId "WWWWWWWW-WWWW-WWWW-WWWW-WWWWWWWWWWWW"
```

Comment:
Similarly, it'd be nice if this was moved to the end of the commands, to match the order of the environment variables above.